src/components/PromiseDetail/PromiseStatusTag.svelte

```diff
@@ -3,15 +3,17 @@
 	import { twMerge } from 'tailwind-merge';
 
 	export let status: PromiseStatus;
+	export let isLarge = true;
```

Comment:
PromiseStatusTag has a story file. You added a new prop, can you update the story to include this too? (May take a look from other tag component story)

Reply:
Good point. Updated.